test/support/capybara.rb

```diff
@@ -21,6 +21,10 @@ def switch_domain(domain)
   xpath {|text| "//tr[./td[text() = '#{text}']]" }
 end
 
+Capybara.add_selector(:row_link) do
```

Comment:
Hmm, why doesn't match?

It may indicate:

1. `:row` selector should be more flexible
2. admin page of pickup_posts has weird DOM